gage_timeseries_gif/src/map_utils.R

```diff
@@ -23,30 +23,26 @@ points_sp <- function(locations){
 
 #' create the sp object 
 #'
-fetch_state_map <- function(...){
-  shift_details <- list(...)
+extract_states <- function(area_name){
```

Comment:
This function could be streamlined a bit more - just a suggestion:

Could map a simple tibble of region, huc, and crs in one 
```
  region_info <- tibble::tibble(
    region = c("AK", "HI", "PR"),
    huc = c("19", "20", "21"),
    crs = c(3338, 6633, 4139)
  )
```

And then filter statement can be reduce:
```
  filtered_sites <- gage_info |> 
    filter(if (area_name == "CONUS") !huc %in% region_info$huc else huc == region_info$huc[region_info$region == area_name])
    
      sites_out <- filtered_sites |> 
    points_sp() |> 
    sf::st_as_sf() |> 
    sf::st_transform(if (area_name == "CONUS") 5070 else region_info$crs[region_info$region == area_name])
  
```